DESCRIPTION

```diff
@@ -50,7 +50,10 @@ Imports:
     dust (>= 0.8.1),
     eigen1,
     mcstate (>= 0.4.0),
-    socialmixr
+    socialmixr,
```

Comment:
dependencies should be sorted alphabetically (with upper case letters sorted before lower case). As discussed below you can and should remove epitrix.

I think that the remaining 2 belong in Suggests as they are not needed for core running of the package. You would remove the importFrom statements, which were not needed anyway as you were using the fully qualified name